datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -3,279 +3,29 @@
-                                                }
-                                            },
-                                        },
-                                        {
-                                            "if": {"properties": {"billing_type": {"pattern": "flat_fee|one_time"}}},
-                                            "then": {
-                                                "not": {
-                                                    "anyOf": [
-                                                        {"required": ["unit_label"]},
-                                                        {"required": ["metric"]},
-                                                        {"required": ["tag"]},
-                                                    ]
-                                                },
-                                                "required": ["unit_price"],
-                                            },
-                                        },
-                                    ],
-                                },
-                            },
-                            "terms": {
-                                "description": "Attributes about terms for an integration",
-                                "type": "object",
-                                "properties": {
-                                    "eula": {
-                                        "description": "A link to a PDF file containing the EULA for this integration",
-                                        "type": "string",
-                                    },
-                                    "legal_email": {
-                                        "description": "Email of the partner company to use for subscription purposes",
-                                        "type": "string",
-                                        "format": "email",
-                                        "minLength": 1,
-                                    },
-                                },
-                                "required": ["eula", "legal_email"],
-                            },
-                        },
-                        "required": ["author", "pricing", "terms"],
-                    },
-                },
-            ],
-            "required": [
-                # Make metric_to_check and metric_prefix mandatory when all integration are fixed
-                'assets',
-                'categories',
-                'creates_events',
-                'display_name',
-                'guid',
-                'integration_id',
-                'is_public',
-                'maintainer',
-                'manifest_version',
-                'name',
-                'public_title',
-                'short_description',
-                'support',
-                'supported_os',
-                'type',
-            ],
-        }
-    )
-
-
-def is_metric_in_metadata_file(metric, check):
```

Comment:
Moved to utils.py https://github.com/DataDog/integrations-core/pull/9111/files#diff-a89ad24f058550876e9ef6f12df02f807ea7117eb7fee1dbf17a012e146fc852R630